app/views/measurements/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<section class="section facilities__form">
```

Comment:
Do these need to be changed to Tailwind actually?

Reply:
Sounds good. Thought it would make sense to be a new branch to go with our other style PRs